app/views/passengers/_form.html.haml

```diff
@@ -19,22 +25,32 @@
     = form.label :phone_number
     = form.phone_field :phone
   .field
+    = form.label :umass_status
+    = form.select :status, Passenger::STATUSES, include_blank: true
+  .field
+    = form.label :registered_with_disability_services
+    = form.check_box :registered_with_disability_services,
+      id: "pax_registered_with_DS"
+  .field
+    = form.label :has_brochure
+    = form.check_box :has_brochure,
+      id: "pax_has_brouchure"
+  .field
     = form.label :mobility_device_id
     = form.select :mobility_device_id, options_from_collection_for_select(MobilityDevice.order(:name), :id, :name), include_blank: 'None'
   - if @current_user.admin?
     .field
       = form.label :active
       = form.check_box :active
-  .field
-    = form.label :permanent_passenger
-    = form.check_box :permanent, disabled: @current_user.dispatcher?,
-      id: "passenger_permanent"
   = form.fields_for :doctors_note, @doctors_note do |d|
     .field
       = d.label :expiration_date
       = d.text_field :expiration_date,
         class: 'datepicker', id: "doctors_note_expiration_date"
   .field
+    = form.label :registering_dispatcher
+    = form.select :registered_by, options_from_collection_for_select(User.all, :id, :name),include_blank: true
```

Comment:
This shouldn't be in the form. It should be done automatically in the controller when a new passenger is created.